data/scripts/install-man-pages.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/env sh
+
+# Determine PREFIX.
+if [ -z "$1" ]; then
+    if [ -n "$PREFIX" ]; then
+        PREFIX='/usr'
+    fi
+else
+    PREFIX="$1"
+fi
+export PREFIX
+
+echo "Installing man pages"
+install -Dm 644 'data/man/tilix' "$PREFIX/share/man/man1/tilix.1"
+gzip -f "$PREFIX/share/man/man1/tilix.1"
+
+if type po4a-translate >/dev/null 2>&1; then
+    for f in data/man/po/*.man.po; do
+        LOCALE="$(basename "$f" .man.po)"
```

Comment:
`$( ... )` statements don't need to be quoted, they already prevent word splitting.

Reply:
Interesting, I didn't know that.